seed/seed.js

```diff
@@ -0,0 +1,9 @@
+const cryptoPairs = {
```

Comment:
We often use names like `constants` for these kinds of data. You don't need a folder. You could just start with a file called `constants.js`. When the project gets bigger the folder can be called `constants` and the file based on the information `currencies.js` or `cryptoPairs.js`